docs/integrations/sumo-apps/sample-data-astronomy.md

```diff
@@ -0,0 +1,58 @@
+---
+id: sumo-logic-sample-data-astronomy
```

Comment:
For Sumo Logic apps URLs, we omit the "sumo-logic-" prefix to keep them clean. See [examples](https://help.sumologic.com/docs/integrations/sumo-apps).

```suggestion
id: sample-data-astronomy
```